.github/workflows/deploy_docs.yaml

```diff
@@ -0,0 +1,45 @@
+name: Deploy mkdocs
+
+on:
+  push:
+    branches:
+      - main
+      - soo-deploy-docs
```

Comment:
@SooluThomas Would you like to keep this? I removed the "run on PR" conditions you said to remove before merge, which I agree with, but wasn't sure about this condition! I can see a good argument either way to keep runs on `soo-deploy-docs`. 

Another nice thing though is that we can trigger a new gh deployment by running `pixi run mkdocs gh-deploy` locally if needed: https://www.mkdocs.org/user-guide/deploying-your-docs/

Reply:
Fixed in https://github.com/NOAA-HES-Capstone/icedyno/pull/5/commits/9d643b50134f606e109c5c54c83a0978091d7b16